controllers/attester_controller.go

```diff
@@ -0,0 +1,104 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	rodev1 "github.com/liatrio/rode/api/v1"
+	"github.com/liatrio/rode/pkg/attester"
+)
+
+// AttesterReconciler reconciles a Attester object
+type AttesterReconciler struct {
+	client.Client
+	Log       logr.Logger
+	Scheme    *runtime.Scheme
+	Attesters []attester.Attester
+}
+
+func (r *AttesterReconciler) ListAttesters() []attester.Attester {
+	return r.Attesters
+}
+
+// +kubebuilder:rbac:groups=rode.liatr.io,resources=attesters,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=rode.liatr.io,resources=attesters/status,verbs=get;update;patch
+
+func (r *AttesterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+	ctx := context.Background()
+	log := r.Log.WithValues("attester", req.NamespacedName)
+	opaTrace := false
+
+	log.Info("Reconciling attester")
+
+	var att rodev1.Attester
+	err := r.Get(ctx, req.NamespacedName, &att)
+	if err != nil {
+		log.Error(err, "Unable to load attester")
+		return ctrl.Result{}, err
+	}
+
+	policy, err := attester.NewPolicy(req.Name, att.Spec.Policy, opaTrace)
```

Comment:
maybe I'm missing something but I don't think we want to compile the policy with every run of the reconcile loop.  we should only want to do it if this has been created for the first time or has been changed.